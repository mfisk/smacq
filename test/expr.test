#!/bin/sh

${BINDIR}smacqq -m <<EOF 
print -v, seq, ipid, seq.double, [ ipid.uint32 * 10000000 ] as foo from pcapfile($TEST/0325@1112-snort.pcap) 
EOF
