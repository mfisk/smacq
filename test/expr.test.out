seq = 3544678043	ipid = 325	seq.double = 3.54468e+09	foo = 3.25e+09
seq = 3544678082	ipid = 326	seq.double = 3.54468e+09	foo = 3.26e+09
seq = 3544678125	ipid = 327	seq.double = 3.54468e+09	foo = 3.27e+09
seq = 3544678126	ipid = 328	seq.double = 3.54468e+09	foo = 3.28e+09
seq = 3593394780	ipid = 329	seq.double = 3.59339e+09	foo = 3.29e+09
seq = 3593444673	ipid = 330	seq.double = 3.59344e+09	foo = 3.3e+09
seq = 3593444674	ipid = 331	seq.double = 3.59344e+09	foo = 3.31e+09
seq = 3593444674	ipid = 332	seq.double = 3.59344e+09	foo = 3.32e+09
seq = 3593444746	ipid = 333	seq.double = 3.59344e+09	foo = 3.33e+09
seq = 3593444883	ipid = 334	seq.double = 3.59344e+09	foo = 3.34e+09
seq = 3593509274	ipid = 335	seq.double = 3.59351e+09	foo = 3.35e+09
seq = 3593571492	ipid = 336	seq.double = 3.59357e+09	foo = 3.36e+09
seq = 3593649239	ipid = 337	seq.double = 3.59365e+09	foo = 3.37e+09
seq = 3593700394	ipid = 338	seq.double = 3.5937e+09	foo = 3.38e+09
seq = 3593755013	ipid = 339	seq.double = 3.59376e+09	foo = 3.39e+09
seq = 3593800672	ipid = 340	seq.double = 3.5938e+09	foo = 3.4e+09
seq = 3593845018	ipid = 341	seq.double = 3.59385e+09	foo = 3.41e+09
seq = 3593509274	ipid = 342	seq.double = 3.59351e+09	foo = 3.42e+09
seq = 3593571492	ipid = 343	seq.double = 3.59357e+09	foo = 3.43e+09
seq = 3593755013	ipid = 344	seq.double = 3.59376e+09	foo = 3.44e+09
seq = 3593800672	ipid = 345	seq.double = 3.5938e+09	foo = 3.45e+09
seq = 3593845018	ipid = 346	seq.double = 3.59385e+09	foo = 3.46e+09
seq = 3593649239	ipid = 347	seq.double = 3.59365e+09	foo = 3.47e+09
seq = 3593700394	ipid = 348	seq.double = 3.5937e+09	foo = 3.48e+09
seq = 3593509274	ipid = 349	seq.double = 3.59351e+09	foo = 3.49e+09
seq = 3593571492	ipid = 350	seq.double = 3.59357e+09	foo = 3.5e+09
seq = 3593755013	ipid = 351	seq.double = 3.59376e+09	foo = 3.51e+09
seq = 3593800672	ipid = 352	seq.double = 3.5938e+09	foo = 3.52e+09
seq = 3593845018	ipid = 353	seq.double = 3.59385e+09	foo = 3.53e+09
seq = 3593649239	ipid = 354	seq.double = 3.59365e+09	foo = 3.54e+09
seq = 3593700394	ipid = 355	seq.double = 3.5937e+09	foo = 3.55e+09
seq = 3593445175	ipid = 356	seq.double = 3.59345e+09	foo = 3.56e+09
seq = 3593445176	ipid = 357	seq.double = 3.59345e+09	foo = 3.57e+09
seq = 3594686715	ipid = 358	seq.double = 3.59469e+09	foo = 3.58e+09
seq = 3594727176	ipid = 359	seq.double = 3.59473e+09	foo = 3.59e+09
seq = 3594727177	ipid = 360	seq.double = 3.59473e+09	foo = 3.6e+09
seq = 3594727177	ipid = 361	seq.double = 3.59473e+09	foo = 3.61e+09
seq = 3594727249	ipid = 362	seq.double = 3.59473e+09	foo = 3.62e+09
seq = 3594727386	ipid = 363	seq.double = 3.59473e+09	foo = 3.63e+09
seq = 3594727588	ipid = 364	seq.double = 3.59473e+09	foo = 3.64e+09
seq = 3594727692	ipid = 365	seq.double = 3.59473e+09	foo = 3.65e+09
seq = 3594727832	ipid = 366	seq.double = 3.59473e+09	foo = 3.66e+09
seq = 3594727895	ipid = 367	seq.double = 3.59473e+09	foo = 3.67e+09
seq = 3594728087	ipid = 368	seq.double = 3.59473e+09	foo = 3.68e+09
seq = 3594728132	ipid = 369	seq.double = 3.59473e+09	foo = 3.69e+09
seq = 3594728232	ipid = 370	seq.double = 3.59473e+09	foo = 3.7e+09
seq = 3594728434	ipid = 371	seq.double = 3.59473e+09	foo = 3.71e+09
seq = 3594728526	ipid = 372	seq.double = 3.59473e+09	foo = 3.72e+09
seq = 3594728626	ipid = 373	seq.double = 3.59473e+09	foo = 3.73e+09
seq = 3594728766	ipid = 374	seq.double = 3.59473e+09	foo = 3.74e+09
seq = 3594728829	ipid = 375	seq.double = 3.59473e+09	foo = 3.75e+09
seq = 3594728997	ipid = 376	seq.double = 3.59473e+09	foo = 3.76e+09
seq = 3594729042	ipid = 377	seq.double = 3.59473e+09	foo = 3.77e+09
seq = 3594729142	ipid = 378	seq.double = 3.59473e+09	foo = 3.78e+09
seq = 3594729282	ipid = 379	seq.double = 3.59473e+09	foo = 3.79e+09
seq = 3594729345	ipid = 380	seq.double = 3.59473e+09	foo = 3.8e+09
seq = 3594729509	ipid = 381	seq.double = 3.59473e+09	foo = 3.81e+09
seq = 3594729661	ipid = 382	seq.double = 3.59473e+09	foo = 3.82e+09
seq = 3594729801	ipid = 383	seq.double = 3.59473e+09	foo = 3.83e+09
seq = 3594729983	ipid = 384	seq.double = 3.59473e+09	foo = 3.84e+09
seq = 3594730147	ipid = 385	seq.double = 3.59473e+09	foo = 3.85e+09
seq = 3594730291	ipid = 386	seq.double = 3.59473e+09	foo = 3.86e+09
seq = 3594730431	ipid = 387	seq.double = 3.59473e+09	foo = 3.87e+09
seq = 3594730565	ipid = 388	seq.double = 3.59473e+09	foo = 3.88e+09
seq = 3594730701	ipid = 389	seq.double = 3.59473e+09	foo = 3.89e+09
seq = 3594730833	ipid = 390	seq.double = 3.59473e+09	foo = 3.9e+09
seq = 3594731049	ipid = 391	seq.double = 3.59473e+09	foo = 3.91e+09
seq = 3594731181	ipid = 392	seq.double = 3.59473e+09	foo = 3.92e+09
seq = 3594731321	ipid = 393	seq.double = 3.59473e+09	foo = 3.93e+09
seq = 3594731455	ipid = 394	seq.double = 3.59473e+09	foo = 3.94e+09
seq = 3594731591	ipid = 395	seq.double = 3.59473e+09	foo = 3.95e+09
seq = 3594731723	ipid = 396	seq.double = 3.59473e+09	foo = 3.96e+09
seq = 3594731939	ipid = 397	seq.double = 3.59473e+09	foo = 3.97e+09
seq = 3594732071	ipid = 398	seq.double = 3.59473e+09	foo = 3.98e+09
seq = 3594732211	ipid = 399	seq.double = 3.59473e+09	foo = 3.99e+09
seq = 3594732345	ipid = 400	seq.double = 3.59473e+09	foo = 4e+09
seq = 3594732481	ipid = 401	seq.double = 3.59473e+09	foo = 4.01e+09
seq = 3594732613	ipid = 402	seq.double = 3.59473e+09	foo = 4.02e+09
seq = 3594732829	ipid = 403	seq.double = 3.59473e+09	foo = 4.03e+09
seq = 3594732961	ipid = 404	seq.double = 3.59473e+09	foo = 4.04e+09
seq = 3594733101	ipid = 405	seq.double = 3.59473e+09	foo = 4.05e+09
seq = 3594733235	ipid = 406	seq.double = 3.59473e+09	foo = 4.06e+09
seq = 3594733371	ipid = 407	seq.double = 3.59473e+09	foo = 4.07e+09
seq = 3594733503	ipid = 408	seq.double = 3.59473e+09	foo = 4.08e+09
seq = 3594733719	ipid = 409	seq.double = 3.59473e+09	foo = 4.09e+09
seq = 3594733851	ipid = 410	seq.double = 3.59473e+09	foo = 4.1e+09
seq = 3594733983	ipid = 411	seq.double = 3.59473e+09	foo = 4.11e+09
seq = 3594734115	ipid = 412	seq.double = 3.59473e+09	foo = 4.12e+09
seq = 3594889131	ipid = 413	seq.double = 3.59489e+09	foo = 4.13e+09
seq = 3594889132	ipid = 414	seq.double = 3.59489e+09	foo = 4.14e+09
seq = 3594889132	ipid = 415	seq.double = 3.59489e+09	foo = 4.15e+09
seq = 3594889204	ipid = 416	seq.double = 3.59489e+09	foo = 4.16e+09
seq = 3594734247	ipid = 417	seq.double = 3.59473e+09	foo = 4.17e+09
seq = 3594734292	ipid = 418	seq.double = 3.59473e+09	foo = 4.18e+09
seq = 3594889360	ipid = 419	seq.double = 3.59489e+09	foo = 4.19e+09
seq = 3594889361	ipid = 420	seq.double = 3.59489e+09	foo = 4.2e+09
seq = 3594734335	ipid = 421	seq.double = 3.59473e+09	foo = 4.21e+09
	ipid = 422		foo = 4.22e+09
	ipid = 41189		foo = 4.1189e+11
seq = 3594734374	ipid = 423	seq.double = 3.59473e+09	foo = 4.23e+09
seq = 3594734666	ipid = 424	seq.double = 3.59473e+09	foo = 4.24e+09
seq = 3594734958	ipid = 425	seq.double = 3.59473e+09	foo = 4.25e+09
seq = 3594735232	ipid = 426	seq.double = 3.59474e+09	foo = 4.26e+09
seq = 3594735506	ipid = 427	seq.double = 3.59474e+09	foo = 4.27e+09
seq = 3594735782	ipid = 428	seq.double = 3.59474e+09	foo = 4.28e+09
seq = 3594736058	ipid = 429	seq.double = 3.59474e+09	foo = 4.29e+09
seq = 3594736332	ipid = 430	seq.double = 3.59474e+09	foo = 4.3e+09
seq = 3594736606	ipid = 431	seq.double = 3.59474e+09	foo = 4.31e+09
